python/ray/util/queue.py

```diff
@@ -28,6 +28,9 @@ class Queue:
     Args:
         maxsize (optional, int): maximum size of the queue. If zero, size is
             unbounded.
+        actor_options (optional, Dict): Dictionary of options to pass into
+            the QueueActor during creation. This could be useful if you
+            need to pass in custom resource requirements, for example.
```

Comment:
Please specifically say that these are the kwargs that can be passed to `Actor.options(`

Reply:
Updated!